users/config.rst

```diff
@@ -206,12 +206,16 @@ path
 type
     Controls how the folder is handled by Syncthing. Possible values are:
 
-    readwrite
+    sendreceive
         The folder is in default mode. Sending local and accepting remote changes.
+        Note that this type was previously called "readwrite" which is deprecated
```

Comment:
Sorry, this flew under my radar. However I'd anyway only merge it once 0.14.49-rc.1 is released, so no rush.

I'd either put the deprecation note in its own paragraph or maybe even use the deprecated directive, something like this

```
    sendonly
        The folder is in "send-only" mode -- it will not be modified by
        Syncthing on this device.
        
        readonly
            .. deprecated:: v0.14.49

            This is the old name of this type, which is still accepted in
            incoming configs.
```